models/models.js

```diff
@@ -1,13 +1,25 @@
 const db = require('../db/connection.js')
+const format = require('pg-format')
 
 exports.selectTopics = () => {
     return db.query('SELECT * FROM topics;').then((result) => {
         return result.rows
     })
 }
 
-exports.selectArticles = () => {
-    return db.query('SELECT articles.article_id, articles.author, title, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC;').then((result) => {
+exports.selectArticles = (topic) => {
+    const validTopics = ['mitch', 'cats', 'paper', undefined]
```

Comment:
We don't want to be hard coding our valid topics, as these can change. Therefore we need to make a query to the database to get all the topics, and then see if our topic is in them (valid)